infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -193,8 +193,8 @@ public SequencerServer(ServerContext serverContext) {
     * @param conflictParam The conflict parameter.
     * @return A conflict hash code.
     */
-    public int getConflictHashCode(UUID streamId, int conflictParam) {
-        return Objects.hash(streamId, conflictParam);
+    public String getConflictHashCode(UUID streamId, int conflictParam) {
+        return streamId.toString() + Integer.toString(conflictParam);
```

Comment:
The hashcode Java uses here doesn't matter what matters is during a hashcode collision equals will be called and the correct conflict param will be used, unlike before. I'll write a test which shows this.